### PR TITLE
Images headers

### PR DESCRIPTION
**Pull request by verwoerd**
This is an combination of pr #4  en #8 . 

I reduced the size of the header slightly, so the text the is more on screen. Fixes padding issues on different size screens. 

 I have not included the text width fix from #8 , since it causes deprecation issues and does not work for all pages.

Some examples:
wide screen
on mobile

@robertdijk @mpsijm Is this something that compromises between both of your PR's?

### CODE REVIEW

**Comment by robertdijk** [comment]
Yea I like it like this

**Comment by robertdijk** [comment]
I think @mpsijm also did something in #8 to fix that the archive page is not so small.

**Comment by verwoerd** [comment]
Nice catch, totally missed that and added a simple css solution to fix it.
image

**Comment by robertdijk** [comment]
I think that besides centering lists, the tables on the archive page should also be wider, and the rest of the text should also be left allighend. I think it's ideal if the archive, news, and front page just look the same as all the other pages. Can we maybe just make the templates of those  _index.md pages more similar to the normal page templates?

**Comment by verwoerd** [comment]
Previous commit should solve that problem. @robertdijk @mpsijm, please let me know if not, else i merge this in and close #4 en #8 .

File: content/archive/_index.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
-title: "Archive"
-description: "The CHipCie Problemset Archive"
+linkTitle: "Archive"
+title: "The CHipCie Problem Set Archive"
```
Comment (mpsijm) [inline content/archive/_index.md:3]: Good change to split up "Problemset" into "Problem Set", thanks :smile: Can you cherry-pick mpsijm@7ad051a623b0baac2e485f213dd17afff3c4e08c? I've applied this change in other places as well :slightly_smiling_face:

**Comment by mpsijm** [comment]
Other than that, I think there's still slightly much space, but it's already better than it was before, so I'll take it :stuck_out_tongue: Thanks again for the fancy images! :smile:

File: content/archive/_index.md (hunk 1 above)
Comment (verwoerd) [reply content/archive/_index.md:3]: Done :)

**Review by mpsijm — APPROVED** [review]

**Review by robertdijk — APPROVED** [review]

**Pull request merged**